src/Microsoft.AspNetCore.Http.Connections/Internal/Transports/LongPollingTransport.cs

```diff
@@ -90,6 +90,8 @@ public LongPollingTransport(CancellationToken timeoutToken, PipeReader applicati
             catch (Exception ex)
             {
                 Log.LongPollingTerminated(_logger, ex);
+                context.Response.ContentType = "text/plain";
```

Comment:
This part of the change conflicts with https://github.com/aspnet/SignalR/commit/3d6e1e69fd03d0aed50f713ece482e7df65434a6. Revert it?

Reply:
Beat me to it, was just updating this